akirachix/Taishi-Informational
# Feature/about

# Engineers Checklist

## What this PR does?

This PR updates the "About" section of the Themis AI informational website.

* What is this? 
  - A content and design update for the "About" page, including the project overview, mission, vision, and values.

* Why this? 
  - To clearly communicate the purpose and goals of Themis AI to website visitors, providing essential information about the project's aims in the Kenyan judicial system.

* How are you doing this? 
  - By implementing a new layout with three key sections (Mission, Vision, Values) and updating the content to reflect Themis AI's objectives.

- List of updates:
  1. Added new "About" section overview text
  2. Created three distinct subsections: Mission, Vision, and Values
  3. Implemented new icons for each subsection
  4. Added responsive layout for better mobile viewing

## Type of change


## How can it be tested?

1. Navigate to the "About" section
2. Verify that all text content matches the approved copy
3. Check that icons are displayed correctly for Mission, Vision, and Values
4. Ensure the layout is responsive and displays correctly on various screen sizes
5. Verify color scheme consistency across the page

![Screenshot from 2024-09-16 15-52-42](https://github.com/user-attachments/assets/7a505fe3-b3cb-4356-80de-3790632a0de5)


## Where is it documented?

- [JIRA ticket] (https://lavenderanyango.atlassian.net/browse/TG-84?atlOrigin=eyJpIjoiMzE1M2E3NzljMDNlNGZhOTlkNTBmNzZlMDQ1ZWRhMDkiLCJwIjoiaiJ9)
